pkg/agent/openflow/pipeline.go

```diff
@@ -2177,13 +2180,16 @@ func (f *featureNetworkPolicy) multiClusterNetworkPolicySecurityDropFlow(table b
 // dnsPacketInFlow generates the flow to send dns response packets of fqdn policy selected Pods to the fqdnController for
 // processing.
 func (f *featureNetworkPolicy) dnsPacketInFlow(conjunctionID uint32) binding.Flow {
-	return AntreaPolicyIngressRuleTable.ofTable.BuildFlow(priorityDNSIntercept).
+	fb := AntreaPolicyIngressRuleTable.ofTable.BuildFlow(priorityDNSIntercept).
 		Cookie(f.cookieAllocator.Request(f.category).Raw()).
-		MatchConjID(conjunctionID).
-		// FQDN should pause DNS response packets and send them to the controller. After
-		// the controller processes DNS response packets, like creating related flows in
-		// the OVS or no operations are needed, the controller will resume those packets.
-		Action().SendToController([]byte{uint8(PacketInCategoryDNS)}, true).
+		MatchConjID(conjunctionID)
+	if f.ovsMetersAreSupported {
+		fb = fb.Action().Meter(PacketInMeterIDNP)
+	}
```

Comment:
While I support adding a meter for this, the meter should be independent from `PacketInMeterIDNP`, which is used for Audit Logging. Otherwise, if a lot of packets needs to be logged, it could lead to a lot of drops of DNS packets, and many issues for workloads. So we should make sure the 2 are independent.

Reply:
Sounds good. Done.